liquibase-core/src/main/java/liquibase/change/core/AlterSequenceChange.java

```diff
@@ -18,12 +19,14 @@
     private String catalogName;
     private String schemaName;
     private String sequenceName;
+    private BigInteger startValue;
```

Comment:
I'm not sure we should be adding the startValue to AlterSequenceChange. Because the sequence already exists, what would be the expected behavior of setting this? Postgresql does have it as a field, but since this change is cross-database I'm worried it will be more confusing than helpful.

Reply:
does it make sense? that's hard question to answer and I don't believe I have one.
why this code was added? because currently we support this functionality for Oracle but, in only Oracle possible way, we drop sequence and recreate it.
to avoid such situation here -- alter used if database allows (to be on the safe side, code is put behind is-Postgres check during AlterSequenceChange creation)

Reply:
We didn't have it in oracle since this attribute wasn't there before. I am always careful about adding new attributes to the change tags since that is the primary interface with end users that we always have to maintain compatibility for and make sure it makes sense.

For postgresql, the startValue doesn't actually change anything at the time, it only changes what the value would rest to if you called it with "restart" without a restart value. 

That feels edge-case-y enough that we should think more about it before adding it to public interface like this.